app/services/suggester/users/recent.rb

```diff
@@ -55,15 +55,21 @@ def recent_commenters(num_comments = 2, limit = 8)
       end
 
       def established_user_article_count
-        Rails.env.production? ? 4 : -1
+        Rails.cache.fetch("established_user_article_count", expires_in: 1.day) do
+          User.average(:articles_count)
+        end
       end
 
       def established_user_comment_count
-        Rails.env.production? ? 4 : -1
+        Rails.cache.fetch("established_user_comment_count", expires_in: 1.day) do
+          User.average(:comments_count)
```

Comment:
this is going to change significantly, as we go from 4 to 1.136, at least for DEV

Reply:
Given this is only used in the onboarding flow I am not very concerned about this value changing from 4 to essentially 2 since its greater than or equal to and counts are whole numbers